packages/compiler/src/aot/summary_serializer.ts

```
@@ -126,9 +127,22 @@ class ToJsonSerializer extends ValueTransformer {
           }
         });
         metadata = clone;
+      } else if (isCall(metadata)) {
+        if (!isFunctionCall(metadata) && !isMethodCallOnVariable(metadata)) {
+          // Don't store complex calls as we won't be able to simplify them anyways lateron.
+          metadata = undefined;
```

Instead of returning `undefined` you should replace it with an `{__symbolic: 'error` }` instead as this might cause a silent failure.

---

Done.